controllers/activemqartemis_controller.go

```diff
@@ -333,12 +333,47 @@ func validateExposeModes(customResource *brokerv1beta1.ActiveMQArtemis) (*metav1
 			return &metav1.Condition{
 				Type:    brokerv1beta1.ValidConditionType,
 				Status:  metav1.ConditionFalse,
-				Reason:  brokerv1beta1.ValidConditionFailedConsoleWithInvalidExposeMode,
+				Reason:  brokerv1beta1.ValidConditionFailedInvalidExposeMode,
 				Message: ".Spec.Console has invalid expose mode route, it is only supported on OpenShift",
 			}, false
 		}
 	}
 
+	for _, acceptor := range customResource.Spec.Acceptors {
+		if acceptor.Expose && (acceptor.ExposeMode != nil && *acceptor.ExposeMode == brokerv1beta1.ExposeModes.Ingress || !isOpenshift) &&
+			customResource.Spec.IngressDomain == "" && acceptor.IngressHost == "" {
+			return &metav1.Condition{
+				Type:    brokerv1beta1.ValidConditionType,
+				Status:  metav1.ConditionFalse,
+				Reason:  brokerv1beta1.ValidConditionFailedInvalidIngressSettings,
+				Message: fmt.Sprintf(".Spec.Acceptors %q has invalid ingress settings, no ingressDomain and ingressHost specified", acceptor.Name),
```

Comment:
... ingressHost unspecified and no Spec.IngressDomain default domain provided"

The ingressHost is part of the acceptor, but the default domain is part of the Spec. I think it is good to point that out.